.github/workflows/build.yaml

```diff
@@ -62,7 +63,15 @@ jobs:
       run: make dist
     # Audit all currently installed packages for security vulnerabilities.
     - name: Audit installed packages
-      run: make audit
+      id: audit-packages
+      continue-on-error: true
+      run: make audit > raw-vulnerabilities.txt && tail -n +5 raw-vulnerabilities.txt > vulnerabilities.txt # Saving the command output and tailing from line 5 to avoid having the command on the PR comment
+    - name: Comment vulnerabilities on PR
+      uses: marocchino/sticky-pull-request-comment@v2
```

Comment:
Can we replace this external action with a call to [`gh issue comment`](https://cli.github.com/manual/gh_issue_comment)?

@eduardo-getpassport I ask because we try to limit our dependencies as much as possible to keep the [software supply chain](https://en.wikipedia.org/wiki/Software_supply_chain) simple.

Reply:
Agreed to use `gh` instead of a 3rd party Action. Also, as mentioned in my previous comment, this can be moved to `pull-request.yaml`.

Reply:
I replaced this with:

`gh pr comment ${{ github.event.number }} --body-file vulnerabilities.txt`

and it works perfectly! thanks for the suggestion!

(pushing the changes after fixing other things)